docs/cli/getting-started.md

```diff
@@ -26,8 +26,10 @@ starboard init
 The `init` subcommand creates the `starboard` namespace, in which Starboard executes Kubernetes jobs to perform
 scans. It also sends custom security resources definitions to the Kubernetes API:
 
-```console
+```
```

Comment:
Was it intentional? To split command and output?

Reply:
Yes! We could leave only the text that It's relevant to copy to the clipboard. And not copy all the output too.